styles.css

```diff
@@ -0,0 +1,294 @@
+body {
+    margin-top: 6em;
+}
+
+section {
+    text-align: center;
+}
+ul {
+    list-style-type: none;
+    padding-inline-start: 0px;
+
+}
+
+h1, h2, h3 {
+    font-family: 'Amatic SC', cursive;
+}
+
+h1 {
+    font-size: 4em;
+    
+}
+
+h2 {
+    font-size: 3em;
+}
+
+h3 {
+    font-size: 2em;
+}
+
+.dark-background {
+    background-color: #1C3150;
+    color: #F2CD60;
+}
+
+p {
+    font-family: monospace;
+    font-size:14px;
+}
+
+.section-header:hover:before, .section-header:hover:after  {
+    content:"⭐";
+}
+.parallax { 
+    height: 500px; 
+    background-attachment: fixed;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+.flex-column-horizontally-centered {
+    display:flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-content: center;
+}
+
+.flex-row-vertically-centered {
+    display:flex;
+    justify-content:space-around;
+    flex-flow:row wrap;
+    align-content: center;
+}
```

Comment:
Good use of class to apply style to different elements of the page. I wonder if there is a way to specify what elements the style applies to?